crates/config/src/config.rs

```diff
@@ -396,6 +396,10 @@ pub struct DipsConfig {
     pub host: String,
     pub port: String,
     pub allowed_payers: Vec<Address>,
+
+    pub price_per_entity: u64,
+    pub price_per_epoch: BTreeMap<String, u64>,
```

Comment:
Use `U256` instead of `u64` with any "wei GRT" value. This will avoid any conversions later.

As a rule of thumb: _prefer an early validation and conversion, over a "scattered" conversion_.